src/web/src/pages/signup.tsx

```diff
@@ -0,0 +1,271 @@
+import Review from '../components/SignUp/Forms/Review';
+import DynamicImage from '../components/DynamicImage';
+
+import { SignUpForm } from '../interfaces';
+import formModels from '../components/SignUp/Schema/FormModel';
+import formSchema from '../components/SignUp/Schema/FormSchema';
+import { usersServiceUrl, webUrl } from '../config';
+
+const {
+  firstName,
+  lastName,
+  displayName,
+  githubUsername,
+  github,
+  githubOwnership,
+  blogUrl,
+  feeds,
+  email,
+  blogOwnership,
+} = formModels;
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '0',
+      margin: '0',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      minHeight: '100vh',
+      width: '100vw',
+      boxSizing: 'border-box',
+      position: 'relative',
+      fontSize: '1.1rem',
+    },
+    imageContainer: {
+      minHeight: '100vh',
+      width: '100vw',
+      position: 'absolute',
+      top: '0',
+      bottom: '0',
+      zIndex: -1,
+      [theme.breakpoints.down(600)]: {
+        display: 'none',
+      },
+    },
+    signUpContainer: {
+      margin: '1% 0 1% 0',
+      display: 'grid',
+      gridTemplateRows: '10% auto 15%',
+      gridGap: '2%',
+      justifyItems: 'center',
+      fontFamily: 'Spartan',
+      height: '510px',
+      width: '510px',
+      padding: '1%',
+      borderRadius: '5px',
+      boxShadow: '2px 4px 4px 1px rgba(0, 0, 0, 0.1)',
+      background: '#ECF5FE',
+      '@media (max-height: 500px) and (max-width: 1024px)': {
+        margin: '0 0 65px 0',
+      },
+      [theme.breakpoints.down(600)]: {
+        background: 'none',
+        boxShadow: 'none',
+        minHeight: '650px',
+        height: '600px',
+        position: 'absolute',
+        top: '0px',
+        width: '100%',
+        margin: '0',
+        padding: '0',
+        gridTemplateRows: '8% auto 17%',
+      },
+    },
+    title: {
+      color: '#121D59',
+      fontSize: '22px',
+    },
+    infoContainer: {
+      width: '100%',
+      position: 'relative',
+    },
+    buttonsWrapper: {
+      margin: '0 auto',
+      width: '50%',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    button: {
+      height: '4rem',
+      width: '40%',
+      fontSize: '1.1em',
+      padding: '0.7em',
+      margin: '5px',
+      background: '#E0C05A',
+      '&:hover': {
+        color: 'black',
+        background: '#EBD898',
+      },
+    },
+    buttonLogin: {
+      height: '4rem',
+      width: '40%',
+      fontSize: '1.1em',
+      padding: '0.7em',
+      margin: '5px',
+      background: '#FF0000',
+      color: '#FFF',
+      '&:hover': {
+        background: '#FF7070',
+      },
+    },
+    text: {
+      textAlign: 'center',
+      fontSize: '0.9em',
+      color: '#474747',
+    },
+  })
+);
+
+const SignUpPage = () => {
+  const classes = useStyles();
+  const [activeStep, setActiveStep] = useState<number>(0);
```

Comment:
Let's use an `enum` to name these steps vs. 0, 1, 2... which are meaningless.

https://www.typescriptlang.org/docs/handbook/enums.html